lib/pg_easy_replicate/orchestrate.rb

```diff
@@ -204,16 +204,19 @@ def switchover(
         group_name:,
         source_conn_string: source_db_url,
         target_conn_string: target_db_url,
-        lag_delta_size: nil
+        lag_delta_size: nil,
+        skip_vacuum_analyze: false
```

Comment:
Mind adding a quick spec for this? You can copy paste this spec [here](https://github.com/shayonj/pg_easy_replicate/blob/ab204be2cd329f5922bd8c3535474b63587c2816/spec/pg_easy_replicate/orchestrate_spec.rb#L485-L499) and expect the last run to be nil when `skip_vacuum_analyze` is passed as true

Happy to follow up otherwise.